src/layout.css

```diff
@@ -4,40 +4,6 @@
  * @section Layout
  */
 
-/*
- * All a limiter to a wrapping element around a page layout. The limiter should never be placed inside other elements that have layout classes. Unlike many other classes in Assembly, the limiter class includes built-in media queries, designed to manage line lengths across screen sizes. The limiter's max width is 1200px.
- *
- * @memberof Layout
- * @example
- * <div>
- *  <div class='limiter border border--darken10 h24'>
- *   limiter
- *  </div>
- * </div>
- */
-.limiter {
-  padding-left: 24px;
-  padding-right: 24px;
-  width: 100%;
-}
```

Comment:
it'd be great to see an example that fully recreates all the details of limiter, across screen sizes, with existing classes.